apps/mappings/tasks.py

```
@@ -261,3 +268,100 @@ def sync_sage_intacct_attributes(sageintacct_attribute_type: str, workspace_id:
+        if (
+            setting.source_field in ['PROJECT', 'COST_CENTER']
+            or setting.is_custom
+        ):
+            task_settings['mapping_settings'].append({
+                'source_field': setting.source_field,
+                'destination_field': setting.destination_field,
+                'destination_sync_methods': [SYNC_METHODS[setting.destination_field]],
+                'is_auto_sync_enabled': True,
+                'is_custom': setting.is_custom,
+                'import_without_destination_id': False,
+                'prepend_code_to_name': True if setting.destination_field in configuration.import_code_fields else False
```

Ruff (0.11.9)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)